cmd/timestamp-server/app/serve.go

```diff
@@ -69,7 +70,12 @@ var serveCmd = &cobra.Command{
 
 		http.Handle("/metrics", promhttp.Handler())
 		go func() {
-			_ = http.ListenAndServe(":2112", nil)
+			srv := &http.Server{
+				Addr:         ":2112",
+				ReadTimeout:  10 * time.Second,
+				WriteTimeout: 10 * time.Second,
+			}
+			_ = srv.ListenAndServe()
```

Comment:
just curious, no error handling here because you don't want to panic?

Reply:
added panic, it should crash on startup